dolphinscheduler-ui/src/views/datasource/list/use-columns.ts

```diff
@@ -28,12 +28,13 @@ import {
 import { EditOutlined, DeleteOutlined } from '@vicons/antd'
 import JsonHighlight from './json-highlight'
 import ButtonLink from '@/components/button-link'
+import { IDataSource, TableColumns } from './types'
```

Comment:
`import type` is better than `import` here, and `import type` should be in the last line.